system-contract-dapp-playground/src/utils/constants.ts

```diff
@@ -0,0 +1,30 @@
+export const HEDERA_SOCIAL_MEDIA = [
+  {
+    name: 'discord',
+    link: 'https://discord.com/invite/hedera',
+  },
+  {
+    name: 'facebook',
```

Comment:
Q: we have a facebook? 😆

Reply:
lool I don't know why but thiss a funny question to me but yeah we have them all. I got these social handles from our site https://hedera.com/. They all are listed in the footer section of the site

Reply:
Yeah you're right. I learned something today 😆